palm/plugins/dbt/commands/cmd_model.py

```diff
@@ -50,9 +51,9 @@ def new(ctx, name: str, model_type: str, use_ref_file: bool):
     #     click.secho(msg, fg="green" if success else "red")
 
 
-def create_model(model_name, model_type, ctx, use_ref_file):
+def create_model(model_name, model_dir, model_type, ctx, use_ref_file):
     model_template_path = Path(Path(__file__).parent.parent, 'model_template', 'model')
-    models_path = Path('models/arbor/', model_type)
+    models_path = Path('models/', model_dir, model_type)
```

Comment:
We have a couple more options here:

* We could remove the `required=True` from the `model-dir` click option, making it optional... then if it's not provided you would have the functionality of dumping everything into `/models`, if it is provided the model will be put in the dir
* Building on that, we could potentially implement a default value for `model-dir` via the `.palm/config`
* We could make `/models` part of the value of `model-dir` which gets us closer to what @emekdahl-palmetto suggested re. not forcing our convention on other people's projects.

Reply:
There's also a place in the `dbt_project.yml` that specifies where the project looks for models... 

I like option 1 or option 3 from Jake's list

Reply:
I'm going to implement #1 for now, as it's the lowest effort and I'd like to get this merged before the next release. If there is value in taking this further we can open a new issue to improve this further

Reply:
Based on the help text for the `--model-dir` flag, I think eliminating "model_type" from the model path is called for. Now we are saying to users, please specify exactly the sub-folder under models you want this model to get generated in. Having one more layer that they didn't ask for is confusing.

Reply:
@mrogers-palmetto - I agree that this should be refined. Since you're more experienced with dbt and have a better idea of what our users would expect, as well as the functionality that we need for our own projects. Would you be interested in taking a shot at making this 'right'? This PR was merged as I wanted to get this fix in for the next release as it meets the initial requirement to remove the hard-coded `arbor` string.

Reply:
Oh snap, I didn't notice it was already merged 😆 
Will do!